k8s/aat/common/divorce/div-cms.yaml

```diff
@@ -14,35 +14,21 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: div-cms
-    version: 1.1.0
+    version: 1.2.0
```

Comment:
The new version allows us to get rid of duplicates by replacing `-aat/-prod` with `{{ .Values.global.environment }}` and move the variables into the common `values.yaml` in each project